examples/auth/index.js

```diff
@@ -1,77 +1,69 @@
-'use strict'
+'use strict';
 
-/**
- * Module dependencies.
- */
-
-var express = require('../..');
-var hash = require('pbkdf2-password')()
-var path = require('path');
-var session = require('express-session');
-
-var app = module.exports = express();
-
-// config
+const express = require('express');
```

Comment:
```suggestion
var express = require('../..');
```

The examples are running as part of the Express tests suite, so express is imported locally and not as an external module